hooks/helmlint.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# We don't set `set -e` in this script because some of the functions will return a non-zero value (e.g contains_element)
+
+# OSX GUI apps do not pick up environment variables the same way as Terminal apps and there are no easy solutions,
+# especially as Apple changes the GUI app behavior every release (see https://stackoverflow.com/q/135688/483528). As a
+# workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
+export PATH=$PATH:/usr/local/bin
+
+# Take the current working directory to know when to stop walking up the tree
+readonly cwd_abspath="$(realpath "$PWD")"
```

Comment:
Is `realpath` available on Mac and Linux by default?

Reply:
Yes. Just checked ubuntu and alpine and it existed so I think it is safe to assume.